services-js/commissions-app/src/pages/commissions/apply.tsx

```diff
@@ -57,7 +57,9 @@ export default class IndexPage extends React.Component<Props> {
               address: Yup.string()
                 .required('Your Address Is Required!')
                 .min(2, 'Your Address Needs To Be Valid'),
-              unit: Yup.string().min(1),
+              unit: Yup.string()
```

Comment:
Looks like you have a regression here. Remember that `unit` shouldn't be required.